packages/ui-components/src/Tooltip/Tooltip.test.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { shallow, mount } from "enzyme";
+
+import Tooltip from "./Tooltip";
```

Comment:
I'll echo Lauren's concern above, that we will need to use CSS Modules here as I think it's important to keep these components completely encapsulated. I am opposed to having any core component rely on any global style. I am checking out your branch to see if I can help investigate the challenges you were having with CSS Modules.